x-pack/plugins/security_solution/public/timelines/containers/details/index.tsx

```diff
@@ -59,13 +61,17 @@ export const useTimelineEventsDetails = ({
   const [loading, setLoading] = useState(false);
   const [timelineDetailsRequest, setTimelineDetailsRequest] =
     useState<TimelineEventsDetailsRequestOptions | null>(null);
+  const [eventDetailsFieldsRepsonse, setEventDetailsFieldsRepsonse] = useState<Record<
```

Comment:
Just renamed this from `rawEventData` to a more descriptive `eventDetailsFieldsResponse`